packages.html

```diff
@@ -221,33 +195,61 @@ <h4>Sample application demonstrating Kitura and IBM Swift features.</h4>
 
           <h3><a href="https://github.com/IBM-Swift/Kubernetes-kitura-sample">Kubernetes-Kitura-Sample</a></h3>
           <h4>Demonstration application using Kubernetes.</h4>
+
+          <h3><a href="https://github.com/IBM-Swift/Kitura-Credentials-Sample">Kitura-Credentials-Sample</a></h3>
+          <h4>Sample web application for authentication using Kitura-Credentials.</h4>
+      </div>
+      <div class="infrastructure card">
+        <img src="assets/infrastructure.png" alt="Infrastructure">
+        <h2>INFRASTRUCTURE</h2>
+          <h3><a href="https://github.com/IBM-Swift/Kitura-net">Kitura-net</a></h3><a href="https://ibm-swift.github.io/Kitura-net/"><h5>API</h5></a>
+          <h4>Logic for sending and receiving HTTP requests.</h4>
+
+          <h3><a href="https://github.com/IBM-Swift/BlueSocket">BlueSocket</a></h3>
+          <h4>Socket framework for Swift.</h4>
+
+          <h3><a href="https://github.com/IBM-Swift/BlueSSLService">BlueSSLService</a></h3>
+          <h4>SSL/TLS add-in framework for BlueSocket.</h4>
+
+          <h3><a href="https://github.com/IBM-Swift/BlueCryptor">BlueCryptor</a></h3>
+          <h4>Cross-platform cryptography library.</h4>
+
+          <h3><a href="https://github.com/IBM-Swift/BlueSignals">BlueSignals</a></h3>
+          <h4>Generic cross-platform signal handler.</h4>
+
+          <h3><a href="https://github.com/IBM-Swift/BlueRSA">BlueRSA</a></h3>
+          <h4>Cross-platform RSA wrapper library.</h4>
+
       </div>
       <div class="misc card">
         <img src="assets/misc.svg" alt="Misc">
         <h2>MISC</h3>
           <h3><a href="https://github.com/IBM-Swift/Swift-SMTP">Swift-SMTP</a></h3>
           <h4>A Swift SMTP mail client.</h4>
 
-        <h3><a href="https://github.com/IBM-Swift/KituraKit">KituraKit</a></h3><a href="https://ibm-swift.github.io/KituraKit/"><h5>API</h5></a>
-          <h4>Client side library for using codable routes with Kitura.</h4>
-
           <h3><a href="https://github.com/IBM-Swift/swift-html-entities">swift-html-entities</a></h3>
           <h4>Pure Swift HTML encode and decode utility tool.</h4>
 
           <h3><a href="https://github.com/IBM-Swift/Swift-JWT">Swift-JWT</a></h3>
-          <h4>An implementation of JSON Web Tokens in pure Swift.</h4>
+          <h4>An implementation of JSON Web Tokens in Swift.</h4>
 
-          <h3><a href="https://github.com/IBM-Swift/Bridging">Bridging</a></h3>
-          <h4>Bridges funtions between macOS and Linux.</h4>
+          <h3><a href="https://github.com/IBM-Swift/CircuitBreaker">CircuitBreaker</a></h3>
```

Comment:
Please leave CircuitBreaker in the Cloud section.

Reply:
Done.